specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/privatelinkhub.json

```diff
@@ -0,0 +1,418 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "schema": {
+              "$ref": "#/definitions/PrivateLinkHubInfoListResult"
+            },
+            "description": "PrivateLinkHub resource details"
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List privateLinkHubs in subscription": {
+            "$ref": "./examples/ListPrivateLinkHubsInSubscription.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ErrorDetail": {
+      "description": "Error details",
+      "type": "object",
+      "properties": {
+        "message": {
+          "description": "Error message",
+          "type": "string"
+        },
+        "code": {
+          "description": "Error code",
+          "type": "string"
+        },
+        "target": {
+          "description": "Error target",
+          "type": "string"
+        }
+      }
+    },
+    "ErrorContract": {
+      "title": "Error details.",
+      "description": "Contains details when the response code indicates an error.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The error details.",
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ErrorResponse"
+        }
+      }
+    },
+    "PrivateLinkHubInfoListResult": {
+      "description": "List of privateLinkHubs",
+      "type": "object",
+      "properties": {
+        "nextLink": {
+          "description": "Link to the next page of results",
+          "type": "string"
+        },
+        "value": {
+          "description": "List of privateLinkHubs",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkHub"
+          }
+        }
+      }
+    },
+    "PrivateLinkHub": {
+      "description": "A privateLinkHub",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "PrivateLinkHub resource properties",
+          "$ref": "#/definitions/PrivateLinkHubProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "PrivateLinkHubPatchInfo": {
+      "description": "PrivateLinkHub patch details",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "description": "Resource tags",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "type": "object"
+        }
+      }
+    },
+    "PrivateLinkHubProperties": {
```

Comment:
why does this type not have any other properties? Can you please provide some more info on what this type does?

Reply:
The resource does not have any properties so far. This resource is going to be used to create a Private Links to Synapse studio, thats all.